pkg/runner/runner.go

```diff
@@ -75,8 +75,12 @@ func (r *JMeterRunner) Run(execution testkube.Execution) (result testkube.Execut
 
 	// Only file based tests in first iteration
 	if execution.Content.IsDir() || !execution.Content.IsFile() {
-		output.PrintLog(fmt.Sprintf("%s Unsupported content type, use file based content", ui.IconCross))
-		return result, fmt.Errorf("unsupported content type, use file based content")
+		output.PrintLog(fmt.Sprintf("%s It is a directory test - trying to find jmx file in %s", ui.IconWorld, path))
+		path, err = r.FindJmxFile(path)
```

Comment:
should we also support explicit argument for the run command, lie we do it for k6?